unison-cli/src/Unison/Codebase/Editor/Output/BranchDiff.hs

```diff
@@ -88,8 +86,16 @@ isEmpty BranchDiffOutput {..} =
 
 type TermDisplay v a = (HashQualified Name, Referent, Maybe (Type v a), MetadataDiff (MetadataDisplay v a))
 
+compareTermDisplay :: TermDisplay v a -> TermDisplay v a -> Ordering
```

Comment:
Why not use an Ord instance here btw?
If this isn't meant to be the canonical ord instance, then maybe add some context to the name.

Reply:
The derived `Ord` instance previously relied on being able to render a `Name` to a string, so it won't do. These comparisons just recover the existing behavior of comparing names alphabetically.

Although - now that you've pointed it out - I think we could just rework the `instance Alphabetical Name` to not rely on a `toText`. It's currently:

```
instance Alphabetical Name where
  compareAlphabetical n1 n2 =
    compareAlphabetical (toText n1) (toText n2)
```

but it'd probably be better to just compare the _segments_' bytes alphabetically. Then I think we could just use the derived `Ord` again.